### PR TITLE
Make JSON library configurable

### PR DESCRIPTION
**Pull request by Hermanverschooten**
I ran into an issue in production, `Poison` was unavailable.
This is of course because Phoenix >= 1.4 defaults to `Jason`.

This change makes the JSON library configurable.

Hope you like it.

### CODE REVIEW

File: lib/recaptcha/http.ex [hunk 1]
```diff
@@ -39,10 +39,11 @@ defmodule Recaptcha.Http do
   def request_verification(body, options \\ []) do
     timeout = options[:timeout] || Config.get_env(:recaptcha, :timeout, 5000)
     url = Config.get_env(:recaptcha, :verify_url, @default_verify_url)
+    json = Application.get_env(:recaptcha, :json_library, Poison)
```
Comment (sobolevn) [inline lib/recaptcha/http.ex]: Is the default value here required? Or we can just have it in the config?

Comment (Hermanverschooten) [reply lib/recaptcha/http.ex]: Whatever you prefer.

Comment (Hermanverschooten) [reply lib/recaptcha/http.ex]: I removed it, you were right.

**Comment by sobolevn** [comment]
Thanks!

**Comment by sobolevn** [comment]
@samueljseay can you please release it?

**Comment by smaximov** [comment]
Run into this issue as well. It works in development, but fails in production with Elixir 1.9 releases because Poison is unavailable. I know I can fetch latest master from Github, but having a hex release would be nice.

**Comment by sobolevn** [comment]
Ping to @samueljseay or @JustMikey 
I need access to `hex` to be able to publish new releases. 
Thanks!

**Comment by samueljseay** [comment]
Hey sorry I've been away from this for ages @sobolevn I will get you hex access if I can.

**Comment by samueljseay** [comment]
@sobolevn you already have hex access as owner afaict. https://hex.pm/packages/recaptcha

**Comment by sobolevn** [comment]
Awesome! Thanks :)

Will release it today! 🎉

**Comment by sobolevn** [comment]
Done https://hexdocs.pm/recaptcha/3.0.0/api-reference.html

**Pull request merged**